index.html

```diff
@@ -40,31 +39,185 @@ <h1 class="heading_second">
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="tech_list">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
           </ul>
+          <button class="btn1" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="work_card">
+        <div class="img_cardiv">
+          <img class="mobi" src="./images/firstimage.png" alt="card" id="mobi-img2" />
+        </div>
+        <div class="side_info">
+          <h3 class="project_name">
+            Multi-Post<br>
+            Stories
+          </h3>
+          <ul class="first_list">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="tech_list">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="btn1" type="submit">See Project</button>
+        </div>
+      </div>
+      <div class="work_card">
+        <div class="img_cardiv">
+          <img class="mobi" src="./images/thirdimage.png" alt="card" id="mobi-img3" />
+        </div>
+        <div class="side_info">
+          <h3 class="project_name">
+            Multi-Post<br>
+            Stories
+          </h3>
+          <ul class="first_list">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="tech_list">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="btn1" type="submit">See Project</button>
+        </div>
+      </div>
+      <div class="work_card">
+        <div class="img_cardiv">
+          <img class="mobi" src="./images/fourthimage.png" alt="card" id="mobi-img4" />
+        </div>
+        <div class="side_info">
+          <h3 class="project_name">
+            Multi-Post<br>
+            Stories
+          </h3>
+          <ul class="first_list">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="tech_list">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="btn1" type="button">See Project</button>
+        </div>
+      </div>
+    </div>
+    <section id="secondlast_section">
+      <div class="section-last-conatiner">
+        <div class="home_containe">
+          <div>
+            <h1 class="aboutdiv">About<br>Myself</h1>
```

Comment:
- [ ] Nice job using `hx` tags for your titles, by the way, keep in mind that only ONE `h1` could exist for each HTML document. In this case you can use an `h2` for the section, following the hierarchy